zerver/views/registration.py

```diff
@@ -354,29 +356,41 @@ def accounts_register(request: HttpRequest) -> HttpResponse:
 
         return login_and_go_to_home(request, auth_result)
 
+    context = {
+        'form': form,
+        'email': email,
+        'key': key,
+        'full_name': request.session.get('authenticated_full_name', None),
+        'lock_name': name_validated and name_changes_disabled(realm),
+        # password_auth_enabled is normally set via our context processor,
+        # but for the registration form, there is no logged in user yet, so
+        # we have to set it here.
+        'creating_new_team': realm_creation,
+        'organisation_type' : organisation_type,
+        'password_required': password_auth_enabled(realm) and password_required,
+        'require_ldap_password': require_ldap_password,
+        'password_auth_enabled': password_auth_enabled(realm),
+        'root_domain_available': is_root_domain_available(),
+        'default_stream_groups': get_default_stream_groups(realm),
+        'accounts': get_accounts_for_email(email),
+        'MAX_REALM_NAME_LENGTH': str(Realm.MAX_REALM_NAME_LENGTH),
+        'MAX_NAME_LENGTH': str(UserProfile.MAX_NAME_LENGTH),
+        'MAX_PASSWORD_LENGTH': str(form.MAX_PASSWORD_LENGTH),
+        'MAX_REALM_SUBDOMAIN_LENGTH': str(Realm.MAX_REALM_SUBDOMAIN_LENGTH)
+    }
+
+    send_email(
+        "zerver/emails/organisation_info", 
+        to_emails=["support@zulipchat.com"],
+        from_name="Zulip Account Security",
+        from_address=FromAddress.tokenized_no_reply_address(),
+        context = context
+    )
+
     return render(
         request,
         'zerver/register.html',
-        context={'form': form,
```

Comment:
If we're going to go this route, can you split out a first preparatory commit that just does the refactor of moving this `context` variable?  

But I'm also not sure we actually want to use the same context for both the email and the page; it seems like we need only like 4 fields for the email.

Reply:
I had made different contexts for different conditions

Reply:
OK, this works.  It'd be nice if you didn't reindent this in the same commit that adds this feature; it obscures what you actually changed here.  Probably best would be to just not re-indent this `context` variable.

Check out our [commit guidelines](https://zulip.readthedocs.io/en/latest/contributing/version-control.html) for more details.